swapi_api/src/modules/characterDetails/characterDetails.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <div>
```

Comment:
miało to być ostylowane w jakikolwiek sposób. na razie to jest wypluty czysty html